### PR TITLE
Map component

### PR DESCRIPTION
**Pull request by Novacci**
In this pull request Map component has been added with styling for dark and light mode
image
image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -15,6 +15,12 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/leaflet@1.9.3/dist/leaflet.css"
+      integrity="sha256-kLaT2GOSpHechhsozzB+flnD+zUyjE2LlfWPgU04xyI="
+      crossorigin=""
+    />
```
Comment (Chriserus) [inline public/index.html:23]: Are you sure we need that, if the dependency is added via package.json? Please check

File: src/Main/Main.module.scss [hunk 2]
```diff
@@ -147,8 +147,10 @@ $font-dark: rgb(138, 148, 158);
 
 .map-container {
   @extend .containter;
-  width: 245px;
-  height: 244px;
+  padding: 0%;
+
```
Comment (Chriserus) [inline src/Main/Main.module.scss]: Unnecessary spacing (enter)

**Review by Chriserus — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (Novacci) [reply public/index.html:23]: Yes, if we get dispose of that line of codes map cant be displayed properly
image

File: src/Main/Main.module.scss (hunk 2 above)
Comment (Novacci) [reply src/Main/Main.module.scss]: Thank you, you've got sharp eye - fixed

**Review by Chriserus — APPROVED** [review]

**Pull request merged**